src/app/facility/analysis/run-analysis/group-analysis/regression-model-selection/regression-model-selection.component.html

```diff
@@ -115,6 +119,10 @@
                         <span *ngIf="model.isValid">Valid</span>
                         <span *ngIf="!model.isValid">Invalid</span>
                     </td>
+                    <td *ngIf="showFailedValidationModel && !model.errorModeling">
+                        <span *ngIf="allModelsValid(model)">Valid</span>
+                        <span *ngIf="!allModelsValid(model)">Invalid</span>
```

Comment:
As a practice we try to avoid binding functions in the template. 

It might be more useful for the user here to *ngFor over the SepValidation object and on each line list:
`predictorVariable: isValid`

Then they will know which predictor is failing validation